tests/pytest/eligibility/test_views.py

```diff
@@ -71,6 +72,39 @@ def model_EnrollmentFlow_with_form_class(mocker, model_EnrollmentFlow):
     return model_EnrollmentFlow
 
 
+@pytest.mark.django_db
+def test_eligibility_logged_in_filtering_flows(mocker, model_TransitAgency, client):
+    model_TransitAgency.pk = None
+    model_TransitAgency.save()
+    digital = models.EnrollmentFlow.objects.create(
+        transit_agency=model_TransitAgency,
+        supported_enrollment_methods=[models.EnrollmentMethods.DIGITAL],
+        label="Digital",
+        selection_label_template="eligibility/includes/selection-label.html",
+    )
+    in_person = models.EnrollmentFlow.objects.create(
+        transit_agency=model_TransitAgency,
+        supported_enrollment_methods=[models.EnrollmentMethods.IN_PERSON],
+        label="In-Person",
+        selection_label_template="eligibility/includes/selection-label.html",
+    )
+    both = models.EnrollmentFlow.objects.create(
+        transit_agency=model_TransitAgency,
+        supported_enrollment_methods=[models.EnrollmentMethods.DIGITAL, models.EnrollmentMethods.IN_PERSON],
+        label="Both",
+        selection_label_template="eligibility/includes/selection-label.html",
+    )
+    model_TransitAgency.save()
+    model_TransitAgency.enrollment_flows.add(digital, in_person, both)
```

Comment:
We don't need this line because when the `EnrollmentFlow`s are being created, they are already setting `model_TransitAgency` as their `transit_agency` (lines 80, 86, and 92).